catalog/justfile

```diff
@@ -63,10 +63,15 @@ up *flags:
 up-deps *flags:
     env COMPOSE_PROFILES="catalog_dependencies" just ../up {{ flags }}
 
+# Load sample data into upstream DB
+init: up
```

Comment:
We would need to wait for the containers to be warmed up, ready and healthy so a `wait-up` recipe might be a better fit as the dependency.

Reply:
No, I didn't get any error, but I assumed that was because we're all on relatively fast computers. I agree with your reasoning though 👍 .